lib/flash_flow/issue_tracker/pivotal.rb

```diff
@@ -74,13 +91,20 @@ def finish(story_id)
       def comment(story_id)
         story = get_story(story_id)
         if story
-          note_prefix = 'Shipped to production on'
           unless already_has_comment?(story, Regexp.new("^#{note_prefix}"))
-            story.notes.create(:text => Time.now.strftime("#{note_prefix} %m/%d/%Y at %H:%M"))
+            story.notes.create(:text => Time.now.strftime(note_time_format))
           end
         end
       end
 
+      def note_prefix
+        'Shipped to production on'
+      end
+
+      def note_time_format
+        "#{note_prefix} %m/%d/%Y at %H:%M"
```

Comment:
I think flash_flow supposed to be general. Specific timezone should be configurable and should be separated story. Putting timezone into output is fine, but all past to this point will be ignored by `flash_flow --release-notes` @bradleyjucsc @ianhall